CHANGELOG.md

```diff
@@ -97,18 +97,19 @@ All notable changes to this project will be documented in this file, in reverse
 
   and removes the dependency http-interop/http-server-middleware.
 
-- [#543](https://github.com/zendframework/zend-expressive/pull/543) renames
+- [#543](https://github.com/zendframework/zend-expressive/pull/543) and
+  [#546](https://github.com/zendframework/zend-expressive/pull/546) renames the
```

Comment:
This change is in `3.0.0alpha2` changelog but milestone for this PR is set to `3.0.0alpha3`.

Reply:
Crap, you're right. I'll update.

Reply:
Essentially, I'll be merging all the alpha logs to a single 3.0.0 log later, but for now, needs to be slightly different.